http/http.go

```diff
@@ -244,6 +284,10 @@ func (s *Server) HealthHandler(w http.ResponseWriter, r *http.Request) *appError
 }
 
 func (s *Server) PortHandler(w http.ResponseWriter, r *http.Request) *appError {
+	err := removeIPParam(r)
+	if err != nil {
+		return badRequest(err).WithMessage(err.Error()).AsJSON()
```

Comment:
Hmm I think it's a bit of an overkill and error prone to rewrite URL and potentially throw a bad request, instead of just having a parameter that tells the function to not look into url 🤔